test/regressions/regression-319-test.coffee

```diff
@@ -0,0 +1,185 @@
+{assert} = require('chai')
+{exec} = require('child_process')
+express = require 'express'
+
+
+PORT = 3333
+
+
+runDredd = (descriptionFile, cb) ->
+  result = {}
+  cmd = "./bin/dredd #{descriptionFile} http://localhost:#{PORT} -ed"
+
+  cli = exec(cmd, (err, stdout, stderr) ->
+    result.exitStatus = err?.code or null
+    result.stdout = '' + stdout
+    result.stderr = '' + stderr
+  )
+  cli.on('close', (code) ->
+    result.exitStatus ?= code if code
+    cb(null, result)
+  )
+
+
+parseOutput = (output) ->
+  # Parse individual entries (deals also with multi-line entries)
+  entries = []
+  entry = undefined
+  for line in output.split(/\r?\n/)
+    match = line.match(/^(\w+): (.+)?$/)
+    if match
+      if entry
+        entry.body = entry.body.trim()
+        entries.push(entry)
+      entry = {label: match[1], body: match[2] or ''}
+    else
+      entry.body += "\n#{line.trim()}"
+
+  # Re-arrange data from entries
+  results = {summary: '', failures: [], bodies: [], schemas: []}
+  for entry in entries
+    switch entry.label
+      when 'body' then results.bodies.push(JSON.parse(entry.body) if entry.body)
+      when 'bodySchema' then results.schemas.push(JSON.parse(entry.body) if entry.body)
+      when 'complete' then results.summary = entry.body
+      when 'fail' then results.failures.push(entry.body)
+  return results
+
+
+describe('Regression: Issue #319', ->
```

Comment:
I'm sorry, I got heavily used to following our [CoffeeScript Style Guide](https://github.com/apiaryio/coffeescript-style-guide) and I didn't realize I should check for an existing style in Dredd. Won't force parentheses here if it doesn't make sense in the context of the existing codebase.